client/components/MessageCondensed.vue

```diff
@@ -5,15 +5,18 @@
 			<span class="from" />
 			<span
 				class="content"
-				@click="onCollapseClick">{{ condensedText }}<button
-					class="toggle-button"
-					aria-label="Toggle status messages" /></span>
+				@click="onCollapseClick"
+			>{{ condensedText }}<button
+				class="toggle-button"
+				aria-label="Toggle status messages"
+			/></span>
```

Comment:
I personally think you get in indentation hell if you allow structures like this. (which is evidenced by the alignment of the buttons' attributes being misindented.)

Maybe the span should end on a newline?

current:
```
			<span
				class="content"
				@click="onCollapseClick"
			>{{ condensedText }}<button
				class="toggle-button"
				aria-label="Toggle status messages"
			/></span>
```

expected:
```
			<span
				class="content"
				@click="onCollapseClick"
			>{{ condensedText }}<button
					class="toggle-button"
					aria-label="Toggle status messages"
				/>
			</span>
```

Reply:
Makes sense, but keep in mind that I didn't make any manual changes, just ran ESLint with `--fix`.
Technically, `<span><button>` and `<span>[newline]</button>` are not the same thing, which is why I believe ESLint went that way. In fact, if you look at our current codebase, we do have both, such as [here](https://github.com/thelounge/thelounge/pull/3083/files#diff-7d16de2e6b8e5d9dd5451e842b5d9034R18), which I think is more in line with what you are suggesting, if I'm not mistaken.